crates/wasmcloud-host/src/host.rs

```diff
@@ -487,13 +512,16 @@ impl Host {
         }
     }
 
-    /// Links are a self-standing, durable entity within a lattice and host runtime. A link
-    /// defines a set of configuration values that apply to an actor and a capability provider indicated
-    /// by the provider's contract ID, public key, and link name. You can set a link before or
-    /// after either the actor or provider are started. Links are automatically established
+    /// Links are a first-class, durable entity within a lattice and host runtime. A link
+    /// defines a set of configuration values that apply to an actor and a capability provider that
+    /// is uniquely keyed by the provider's contract ID, public key, and link name.
+    ///
+    /// You can set a link before or after either the actor or provider are started. Links are automatically established
     /// when both parties are present in a lattice, and re-established if a party temporarily
-    /// leaves the lattice and rejoins (which can happen during a crash or a partition event). Link
-    /// data is exactly as durable as your choice of lattice cache provider
+    /// leaves the lattice and rejoins (which can happen during a crash or a network partition event). The resiliency and
+    /// reliability of link definitions within a lattice are inherited from your choice of lattice cache provider.
+    ///
+    /// The value of `actor` for a link definition can either be the actor's public key _or_ an OCI reference URL.
```

Comment:
For this, you can specifically annotate the argument like so:
```
/// Links links blah blah resilient lattice
/// 
/// * `actor` - Actor public key _or_ OCI reference URL
fn set_link (actor: &str) {}
```
Unfortunately it won't show live until we republish interfaces, but here's what it looks like with the logging interface:
![image](https://user-images.githubusercontent.com/12040685/114452021-f529d280-9ba5-11eb-8362-a1deeeeb3945.png)

Reply:
This kind of annotation may also help out for any other arguments that are named `provider_ref`, `actor_id`, etc

Reply:
`links links blah blah resilient lattice` feels like our new marketing tagline.

Reply:
`r/wasmcloud_circlejerk` incoming